GoogleVision/src/main/java/org/example/GoogleVision.java

```diff
@@ -0,0 +1,164 @@
+package org.example;
+
+import com.google.cloud.vision.v1.*;
+import com.google.protobuf.ByteString;
+
+import java.io.FileInputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GoogleVision {
+    String orderedText = "";
+    public GoogleVision(String filePath) throws Exception{
```

Comment:
i think we should move these business logic out of the constructor generally we do not want our constructors to do any real work. we can move them to a method instead